server/lib/notifications/__mocks__/notification-service.ts

```diff
@@ -0,0 +1,17 @@
+const retrieveNotifications = jest.fn().mockResolvedValue([])
```

Comment:
I'm not a fan of doing this, it obscures the fact that this is a mock in tests and also makes it impossible to inspect the mock without a cast, would *vastly* prefer you either

a) mock directly in the test file
or
b) write a fake in a file that is not here and does not automatically replace the import. With injection, using mocked imports should be unnecessary

This file also makes zero references to the original types, and thus if it gets out of sync we will never know unless the test fails, and that's really not great.